packages/audiodocs/docs/guides/see-your-sound.mdx

```diff
@@ -0,0 +1,262 @@
+import {
+  AudioContext,
+  AudioBuffer,
+  AudioBufferSourceNode,
+} from 'react-native-audio-api';
+import { ActivityIndicator, View, Button, LayoutChangeEvent } from 'react-native';
+
+const AudioVisualizer: React.FC = () => {
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const audioContextRef = useRef<AudioContext | null>(null);
+  const bufferSourceRef = useRef<AudioBufferSourceNode | null>(null);
+  const audioBufferRef = useRef<AudioBuffer | null>(null);
+
+  const handlePlayPause = () => {
+    if (isPlaying) {
+      bufferSourceRef.current?.stop();
+    } else {
+      if (!audioContextRef.current) {
+        return
+      }
+
+      bufferSourceRef.current = audioContextRef.current.createBufferSource();
+      bufferSourceRef.current.buffer = audioBufferRef.current;
+      bufferSourceRef.current.connect(audioContextRef.current.destination);
+
+      bufferSourceRef.current.start();
+    }
+
+    setIsPlaying((prev) => !prev);
+  };
+
+  useEffect(() => {
+    if (!audioContextRef.current) {
+      audioContextRef.current = new AudioContext();
+    }
+
+    const fetchBuffer = async () => {
+      setIsLoading(true);
+      audioBufferRef.current = await FileSystem.downloadAsync(
+        'https://software-mansion-labs.github.io/react-native-audio-api/audio/music/example-music-02.mp3',
+        FileSystem.documentDirectory + 'audio.mp3'
+      ).then(({ uri }) => {
+        return audioContextRef.current!.decodeAudioDataSource(uri);
+      });
+
+      setIsLoading(false);
+    };
+
+    fetchBuffer();
+
+    return () => {
+      audioContextRef.current?.close();
+    };
+  }, []);
+
+  return (
+    <View style={{ flex: 1}}>
+      <View style={{ flex: 0.2 }} />
+      <View
+        style={{ flex: 0.5, justifyContent: 'center', alignItems: 'center' }}>
+        {isLoading && <ActivityIndicator color="#FFFFFF" />}
+        <View
+          style={{
+            justifyContent: 'center',
+            flexDirection: 'row',
+            marginTop: 16,
+          }}>
+        <Button
+            onPress={handlePlayPause}
+            title={isPlaying ? 'Pause' : 'Play'}
+            disabled={!audioBufferRef.current}
+            color={'#38acdd'}
+          />
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default AudioVisualizer;
+```
+
+## Create an analyzer to capture and process audio data
+
+To obtain frequency and time-domain data, we need to utilize the [`AnalyserNode`](/visualization/analyser-node).
+It is an [`AudioNode`](/core/audio-node) that passes data unchanged from input to output while enabling the extraction of this data in two domains: time and frequency.
+We will use two specific [`AnalyserNode's`](/visualization/analyser-node) methods: [`getByteTimeDomainData`](/visualization/analyser-node#getbytetimedomaindata) and [`getByteFrequencyData`](/visualization/analyser-node#getbytefrequencydata).
```

Comment:
As the link is provided in the first sentence, we can skip it here. (This is the wikipedia-way 🙂)
```suggestion
We will use two specific `AnalyserNode's` methods: [`getByteTimeDomainData`](/visualization/analyser-node#getbytetimedomaindata) and [`getByteFrequencyData`](/visualization/analyser-node#getbytefrequencydata).
```

Reply:
Can you make the methods to be a short bullet list?